src/azure-cli/azure/cli/command_modules/acs/_help.py

```diff
@@ -158,6 +158,8 @@
 parameters:
   - name: --acr
     short-summary: The FQDN of the ACR.
+  - name: --hostname
```

Comment:
with second thought, I think it's better rename as `node-name` which is more consistent with other aks command parameters

Reply:
Sure, will do.